model/validation.go

```diff
@@ -669,3 +661,89 @@ func ValidateDestinationPolicy(msg proto.Message) error {
 
 	return errs
 }
+
+func validateProxyAddress(hostAddr string) error {
```

Comment:
The snippet above returns an error if the host is IPv6. Anything else (IPv4 or DNS name) should be okay. I believe that is what you wanted to validate?

See https://play.golang.org/p/zsWMAoutrB